SU-HSDO/suhumsci
# SHS-5683: Blockquote implementation

# READY FOR REVIEW

## Summary
Update blockquotes in wysiwygs

## Need Review By (Date)
07/12

## Urgency
medium

## Steps to Test
1. On Colorful and Traditional
2. Create a Flexible Page
3. Add a `Text Area` component and add a `Blockquote`
4. Check it does look according to what's in Option A in [Figma](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1384-56&t=YEcvFLF7ZsYur3nv-4)
5. Add a `Testimonial` component too to confirm it still look the same
6. Please test in all the color variants in both themes
7. Also check the blockquote block in footer (normal and dark) looks good
8. Add blockquotes in the following pages to confirm they look good:

- https://hs-traditional.suhumsci.loc/news/jackelyn-hwang-wins-jane-addams-award-best-article-community-and-urban-sociology-section
- https://hs-traditional.suhumsci.loc/people/abhinanda-sarkar
- https://hs-traditional.suhumsci.loc/publications/privilege-and-punishment-how-race-and-class-matter-criminal-court
- https://hs-traditional.suhumsci.loc/components/text-area-typography

9. Check VRT https://percy.io/da667d23/suhumsci/builds/35211009 for more testing

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207762326049451
  - https://app.asana.com/0/0/1207792433158565
